packages/office-ui-fabric-react/src/components/Dropdown/examples/Dropdown.Required.Example.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Dropdown, DropdownMenuItemType } from 'office-ui-fabric-react/lib/Dropdown';
+import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
+
+export class DropdownRequiredExample extends BaseComponent<{}, {}> {
+  constructor(props: {}) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div className="docs-DropdownExample">
+        <Dropdown
+          placeholder="Select an Option"
+          label="Required dropdown example:"
+          id="Requireddrop1"
```

Comment:
Is there a reason we need explicit IDs on the examples? If not, might be better to remove it and just let the component auto-generate IDs for the label/field--I noticed a problem with the ComboBox examples last week where examples got copy-pasted (then slightly modified) but all had the same ID.

Reply:
Not sure we do, I just followed the pattern in the other example files, all of which specify the ids. I can remove this one though!